cdap-ui/app/features/adapters/adapters.less

```diff
@@ -14,42 +14,523 @@
+
+      .left-panel-wrapper {
+        .left-panel {
+          height: 100%;
+        }
+      }
+    }
+    my-dag {
+      width: 100%;
+      // Dag-Height 60% on smaller screen
+      height: 60vh;
+      @media (min-height: 800px) {
+        height: 40vh;
+      }
+    }
+
+    .tab-pane .panel-default {
+      .panel-heading > span.panel-title { line-height: 30px; }
+    }
+
+    .as-sortable-drag {
+      &.template {
+        &.transforms {
+          border: 1px dashed black;
+          color: black;
+          background-color: white;
+          text-align: center;
+          font-size: 18px;
+        }
+      }
+    }
+
+    .plugins {
+      h4 { margin-top: 0; }
+      .panel.panel-default {
+        margin-bottom: 5px;
+        .panel-body{
+          padding: 5px;
+
+          .list-group {
+            margin-bottom: 0;
+            height: 150px;
+            overflow: scroll;
+
+            .list-group-item {
+              border: 0;
+              border-bottom: 1px solid lightgray;
+              background-color: transparent;
+              cursor: pointer;
+
+              &:last-child { border: 1px solid transparent; }
+
+              .description-overflow {
+                white-space: nowrap;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                color: darkgray;
+                font-size: 11px;
+              }
+            }
+            li.list-group-item:hover {
+              background-color: @brand-primary;
+              .fa-info-circle {
+                color: white;
+                padding-left: 2px;
+              }
+              .description-overflow {
+                color: black;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    .col-xs-12.text-right {
+      > .btn.btn-default { margin-bottom: 20px; }
+    }
+
+    .panel.panel-default {
+      .panel-heading {
+        cursor: pointer;
+        .btn { border: 0; }
       }
     }
-  }
 
-  .name-error {
-    background-color: @brand-danger;
+    .fa-asterisk {
+      font-size: 9px;
+      color: @brand-danger;
+    }
   }
 
+  &.state-adapters-list {
+    main.container {
+      width: 100%;
+      padding-right: 0;
+      padding-left: 0;
+    }
+    .side-panel.top {
+      background-color: white;
+      font-weight: 500;
+      position: fixed;
+      top: 60px;
+      right: 0;
+      left: 0;
+      z-index: 1028;
+      height: 60px;
+      .box-shadow(0 7px 7px fade(black, 10%), 0 19px 59px fade(black, 25%););
+      h1 {
+        border-right: 1px solid @table-border-color;
+        display: inline-block;
+        font-size: 20px;
+        line-height: 60px;
+        margin: 0;
+        padding: 0 15px;
+      }
+      > div {
+        padding: 7px 15px;
+        span {
+          display: block;
+          &:first-child {
+            font-size: 15px;
+          }
+          &:last-child {
+            font-size: 18px;
+          }
+        }
+      }
+      ul {
```

Comment:
I think the <ul> and <li> are for the total count on the status bar. @astewes correct me if I'm wrong?